.github/pull_request_template.md

```diff
@@ -0,0 +1,58 @@
+## Description
+
+Describes the nature of your changes. If they are substantial, you should
+further subdivide this into a section describing the problem you are solving and
+another describing your solution.
```

Comment:
Let's put it into `<!---     -->`, then GitHub won't render it. It's relevant for the person writing PR description, but not for the person reading it.
Same for the `Related issues(s)` section.

Reply:
Well I thought the person writing the PR would replace these with their description.
But sure. I can do that.

Reply:
Yes, sure. I think it's just a bit quicker to type description below than to select this text and write description replacing this text. Saves 2 seconds, basically :sweat_smile:

Reply:
I don't feel strongly either way, so I am happy to oblige. :smiley: